src/components/Reports/FourteenMonthReports/Layout/Table/Table.tsx

```diff
@@ -140,6 +141,31 @@ export const FourteenMonthReportTable: React.FC<
                           <Link>{contact.name}</Link>
                         </NextLink>
                       </NameTypography>
+                      {(contact.lateBy30Days || contact.lateBy60Days) && (
+                        <Tooltip
+                          title={t('{{daysLate}}+ days late', {
+                            daysLate: contact.lateBy60Days ? '60' : '30',
+                          })}
+                          arrow
+                        >
+                          <Box
+                            sx={{
```

Comment:
Since this is a rather complicated `sx` block, can we make this a styled component? For the `lateBy60Days` flag, you could either pass in a prop or add/remove a `60-days-late` classname and add custom styles when that classname is present. Something like this in the styled component.

```
bgcolor: 'orange',
'&.60-days-late': {
  bgcolor: 'red',
},
```